src/REVBasicDeployer.sol

```diff
@@ -328,17 +353,20 @@ contract REVBasicDeployer is ERC165, IREVBasicDeployer, IJBRulesetDataHook, IERC
         virtual
         returns (uint256 revnetId)
     {
+        (JBRulesetConfig[] memory rulesetConfigurations, bytes memory encodedConfiguration) =
+            _makeRulesetConfigurations(configuration, address(dataHook), extraHookMetadata);
```

Comment:
We need to give the terminals access to the surplus allowance so that the suckers can use it

Reply:
good call